modules/odc-monitoring-your-project-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+[id="odc-monitoring-your-project-metrics_{context}"]
+= Monitoring your project metrics
+
+After you create applications in your project and deploy them, you can use the *Developer* perspective in the web console to see the metrics for your project.
+On the left navigation panel of the *Developer* perspective, click *Monitoring* to see the *Dashboard*, *Metrics*, and *Events* for your project.
+
+.Procedure
+
+* Use the *Dashboard* tab to see graphs depicting the CPU, memory, and bandwidth consumption and network related information, such as the rate of transmitted and received packets and the rate of dropped packets.
++
+image::odc_project_dashboard.png[Monitoring dashboard]
```

Comment:
For each of the screenshots, consider adding an image title:

```
.Image title
image::file.png[Alt text]
```

Reply:
That's a good suggestion, I will need to do this for all the screenshots in the ODC docs to keep it consistent. If you don't mind I will do that as a separate PR. I have created a Jira issue here: https://issues.redhat.com/browse/RHDEVDOCS-1736